src/main/kotlin/no/skatteetaten/aurora/boober/feature/Feature.kt

```diff
@@ -227,7 +227,7 @@ class HeaderHandlers private constructor(defaultAppName: String, defaultEnvName:
                 validator = {
                     it.allowedPattern(
                         "^[a-z][-a-z0-9]{0,38}[a-z0-9]$",
-                        "Name must be alphanumeric and no more than 40 characters",
+                        "The first character of name must be a letter [a-z] and the last character must be alphanumeric. Total length needs to be no more than 40 characters. The rest of the characters can be alphanumeric or a hyphen.",
```

Comment:
Dette er jo ikke bare enkelt.  Jeg går ut fra at vi snakker noe i nærheten av kebab-case her.  Den lange beskrivelsen speiler jo godt regexp, men er regexp korrekt ut fra hva man egentlig ønsker?  Skal det for eksempel være lov med to påfølgende `-`?   Denne er f.eks. helt gyldig ift regexp, men kanskje ikke ift intensjonen: an-example--0f-s0m37h1n6-1--2---3

Reply:
Hehe, helt enig. Men om man tenker på hva som er sannsynlige feil, så kommer vi ganske langt med dette. Kubernetes vil jo evt stoppe ting litt senere i prosessen.

Det er jo ikke HELT klart hva vi vil heller, siden kravene er forskjellige for ulike ressurs-typer... Men, la ikke det beste være det godes fiende! Jeg synes dette er en forbedring, i alle fall! :-)